File: laramie-importacion-webapi/src/domain/entities/etiqueta.ts
export default class Etiqueta {

    id: number;
	entidad: string;
	idEntidad: number;
	codigo: string;

	constructor(
        id: number = 0,
		entidad: string = "",
		idEntidad: number = 0,
		codigo: string = ""
	)
	{
        this.id = id;
		this.entidad = entidad;
		this.idEntidad = idEntidad;
		this.codigo = codigo;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.entidad = row.entidad ?? "";
		this.idEntidad = row.idEntidad ?? 0;
		this.codigo = row.codigo ?? "";
	}

}

File: modules/tesoreria/models/etiqueta.py
from sqlalchemy import Column, BigInteger, String

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class Etiqueta(Base):
    __tablename__ = "tesoreria_etiquetas"

    id = Column(BigInteger, primary_key=True)
    entidad = Column(String(50), nullable=False, index=True)
    id_entidad = Column(BigInteger, nullable=False, index=True)
    codigo = Column(String(50), nullable=False)
